src/components/Application.pcss

```diff
@@ -96,11 +95,13 @@ details {
         white-space: pre-wrap;
     }
 
+    li {
```

Comment:
Styles for navigation list should be in `src/components/TreeMenu/parts/Details/Details.pcss`. Please, use global styles with carefully.